### ISSUE TITLE
No instance configuration provided directly.

### ISSUE THREAD

**Issue by K-945** [kaiwaehner/hivemq-mqtt-tensorflow-kafka-realtime-iot-machine-learning-training-inference#38]
hi,friend.I use docker to build the simulator.I changed the address in scenario.xml,and then i ran build-generator.sh.When i construct a mirror images of simulator ,i run the container

,but 'No instance configuration provided directly.attempting dynamic discover method.'appears.

![0)P2@6$}~F {6W2T}DRF_`K](https://user-images.githubusercontent.com/48827771/77129204-13e69400-6a8e-11ea-9581-7a5d1bc830f1.png)

**Comment by sbaier1** [comment]
Hey, the load generator is intended for use with the [Kubernetes CLI](https://github.com/kaiwaehner/hivemq-mqtt-tensorflow-kafka-realtime-iot-machine-learning-training-inference/blob/master/infrastructure/test-generator/kube-cli.sh) in this repository, as it parameterizes the container properly.
The load generator also uses the Kubernetes API to provision its workers internally, so running it on K8s is pretty much necessary.

Bottom line: nothing is wrong with your build, you just need to run it as intended :)

**Comment by K-945** [comment]
Thanks a million.I have a try.
